plugins/LanguagesManager/Commands/FetchFromTransifex.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\LanguagesManager\Commands;
+
+use Piwik\Container\StaticContainer;
+use Piwik\Plugins\LanguagesManager\API as LanguagesManagerApi;
+use Piwik\Translation\Transifex\API;
+use Symfony\Component\Console\Helper\ProgressHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ */
+class FetchFromTransifex extends TranslationBase
```

Comment:
Since the command is `translations:fetch`, maybe it would make more sense to keep the class name generic too? (e.g. `UpdateTranslations`?) That way it would avoid the situation here where `FetchFromOtrance` is deleted and `FetchFromTransifex` is created whenever we change again from provider?